### PR TITLE
add information about the --publisher flag for aar

### PR DESCRIPTION
**Pull request by ru-fu**
Signed-off-by: Ruth Fuchss <ruth.fuchss@canonical.com>

### CODE REVIEW

**Review by morphis — APPROVED** [review]
LGTM, thanks

File: howto/aar/configure.md [hunk 1]
```diff
@@ -106,13 +106,21 @@ Use "aar trust [command] --help" for more information about a command.
 
 Every AMS instance has a registry-specific client certificate that is stored at `/var/snap/ams/common/registry/client.crt`. To manually register an AMS client, you'll need to copy this certificate to the machine hosting AAR and use the CLI to trust it.
 
-Use any of the following commands to do that:
+To add an AMS client as trusted [publisher](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```
Comment (aguadoenzo) [inline howto/aar/configure.md:109]: Shouldn't it be
```suggestion
To add an AMS client as a trusted [publisher](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```

File: howto/aar/configure.md [hunk 2]
```diff
@@ -106,13 +106,21 @@ Use "aar trust [command] --help" for more information about a command.
...
 
-    cat client.crt | sudo aar trust add
+    sudo aar trust add client.crt --publisher
+
+To add an AMS client as trusted [client](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```
Comment (aguadoenzo) [inline howto/aar/configure.md:113]: Same here
```suggestion
To add an AMS client as a trusted [client](https://discourse.ubuntu.com/t/application-registry/17761#aar-roles), use the following command:
```

**Review by aguadoenzo — APPROVED** [review]
Minor nitpick, otherwise LGTM

File: howto/aar/configure.md (hunk 1 above)
Comment (ru-fu) [reply howto/aar/configure.md:109]: That's the thing I always do wrong ... yes, it should be. :) Thanks!

**Pull request merged**